alert-common/src/main/java/com/synopsys/integration/alert/common/util/DateUtils.java

```diff
@@ -24,38 +24,46 @@
 
 import java.text.ParseException;
 import java.text.SimpleDateFormat;
+import java.time.OffsetDateTime;
 import java.time.ZoneOffset;
-import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
 import java.util.Date;
-import java.util.TimeZone;
 
 public final class DateUtils {
     public static final String DOCKER_DATE_FORMAT = "yyyy-MM-dd'T'HH:mm:ss.SSSSSS'Z'";
     public static final String AUDIT_DATE_FORMAT = "yyyy-MM-dd HH:mm:ss.SSS";
     public static final String UTC_DATE_FORMAT_TO_MINUTE = "yyyy-MM-dd HH:mm '(UTC)'";
 
-    public static Date createCurrentDateTimestamp() {
-        final ZonedDateTime zonedDateTime = ZonedDateTime.now().withZoneSameInstant(ZoneOffset.UTC);
-        return Date.from(zonedDateTime.toInstant());
+    public static OffsetDateTime createCurrentDateTimestamp() {
+        return OffsetDateTime.now();
     }
 
     public static String createCurrentDateString(String format) {
-        Date currentDate = createCurrentDateTimestamp();
+        OffsetDateTime currentDate = OffsetDateTime.now();
```

Comment:
Shouldn't this still call `createCurrentDateTimestamp`?

Reply:
I probably should have deprecated that method. I will do that now.

Reply:
After a quick offline discussion, it makes more sense to keep this method and replace the `OffsetDateTime.now()` to minimize type migration headaches in the future.